osidb/tests/endpoints/flaws/test_flaws.py

```diff
@@ -1819,3 +1820,64 @@ def test_flaw_history_no_history(self, auth_client, test_api_uri):
         body = response.json()
         flaw_json = body["results"][0]
         assert "pgh_diff" not in flaw_json["history"]
+
+    @pytest.mark.parametrize(
+        "workflow_status, embargoed, internal",
+        [
+            (WorkflowModel.WorkflowState.NEW, True, False),
+            (WorkflowModel.WorkflowState.NEW, False, False),
+            (WorkflowModel.WorkflowState.NEW, False, True),
+            (WorkflowModel.WorkflowState.DONE, True, False),
+            (WorkflowModel.WorkflowState.DONE, False, False),
+            (WorkflowModel.WorkflowState.DONE, False, True),
+        ],
+    )
```

Comment:
fun fact: parametrize creates matrices and are thus multiplicative, this is equivalent to:

```python
@pytest.mark.parametrize("workflow_status", (NEW, DONE))
@pytest.mark.parametrize("embargoed, internal", [(True, False), (False, False), (False, True)])
```